idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -331,97 +339,99 @@ def fetch_genome_work(s3genome, strict):
                            "hg38_pantro5_k16"):
         write_to_log("Oh hello interesting new genome {}".format(genome_name))
     genome_dir = os.path.join(REF_DIR, genome_name)
+
     if not os.path.exists(genome_dir):
         try:
-            #TODO Clean up and fold into fetch_reference
-            # Hmm. I understand the desire to create a single function for fetching any reference we might need,
-            # but the functionality of these two functions is actually quite different.  One of them handles
```

Comment:
You seem to be deleting these comments without actually implementing what they are suggesting. Is this because we no longer think the recommendation in the TODO is valid?

Reply:
I took this out b/c the "Hmm" part was Boris responding to the TODO (to not combine with fetch_reference), and it seems like they do do pretty different things so I don't think we should combine them. But I could leave it there?

Reply:
Yeah this was a discussion between YF and me in the comments.   I am not sure if we need it or not need it;  but if code cleanliness is an objective, we could record the discussion in an issue and refer to it here, shortening the comment.